packages/gatsby-transformer-sharp/src/customize-schema.js

```diff
@@ -73,10 +74,10 @@ const fixedNodeType = ({
             }),
         },
         aspectRatio: { type: GraphQLFloat },
-        width: { type: GraphQLFloat },
-        height: { type: GraphQLFloat },
-        src: { type: GraphQLString },
-        srcSet: { type: GraphQLString },
+        width: { type: new GraphQLNonNull(GraphQLFloat) },
+        height: { type: GraphQLNonNull(GraphQLFloat) },
+        src: { type: GraphQLNonNull(GraphQLString) },
+        srcSet: { type: GraphQLNonNull(GraphQLString) },
```

Comment:
Can you change it to `new GraphQLNonNull` everywhere for consistency (height, src, srcSet)?

Reply:
Yes, sure. Just changed it with my last commit.